### PR TITLE
Breadcrumb fix

### PR DESCRIPTION
**Pull request by y-lohse**
I deleted a variable that seemed unused, but the file was `required` elsewhere, and stylus doesn't complain about undefined variables.

### CODE REVIEW

**Comment by y-lohse** [comment]
[iOS+Android 1.18.3 : Regression visuelle fil d'ariane](https://trello.com/c/RSWPnw2E/1974-iosandroid-1183-regression-visuelle-fil-dariane)

File: src/drive/web/modules/navigation/breadcrumb.styl [hunk 1]
```diff
@@ -32,15 +31,15 @@
     height     $coz-bar-size
     border     0
     flex-shrink 0
-    background embedurl('../assets/icons/icon-arrow-left.svg') center center no-repeat
+    background embedurl('../../../assets/icons/icon-arrow-left.svg') center center no-repeat
```
Comment (Crash--) [inline src/drive/web/modules/navigation/breadcrumb.styl]: Why this change.

Comment (y-lohse) [reply src/drive/web/modules/navigation/breadcrumb.styl]: This is so weird. Once I removed `@require '../../../styles/topbar.styl'`, the build started failing… because indeed theses pathes were wrong. Why do they work when the `@require '../../../styles/topbar.styl'` line is present is beyond me.
